content/docs/dvclive/user-guide/quickstart.md

```diff
@@ -0,0 +1,98 @@
+# Quickstart
+
+DVCLive is a simple Python library whose interface consists of three main
+methods.
+
+## Steps
+
+To get it up and running you just need to follow these steps:
+
+### 1. Initialize DVCLive
```

Comment:
Do we even need this in the quickstart, especially if we aren't providing any options?

Reply:
I'm not sure if I like the pattern of omiting `dvclive.init` everywhere (i.e. none of the integrations or examples use it). Even if it currently does nothing, it might be benefitial to promote it's usage as it's a single line.

For me, it already feels a little strange the pattern of implicitly calling `init` on the first `dvclive.log`. Although that might deserve a separate discussion on the `dvclive` side.